cdc/log.cc

```diff
@@ -1320,6 +1291,28 @@ class transformer final : public change_processor {
         return _touched_parts;
     }
 
+    bytes_opt get_col_from_row_state(const cell_map* state, const column_definition& cdef) const {
+        if (state) {
+            if (auto it = state->find(&cdef); it != state->end()) {
+                return it->second;
+            }
+        }
+        return std::nullopt;
+    }
+
+    cell_map* get_current_row_state(const clustering_key* ck) {
+        if (!ck) {
+            // static row case
+            return &_static_row_state;
+        } else {
+            auto it = _clustering_row_states.find(*ck);
+            if (it != _clustering_row_states.end()) {
```

Comment:
Suggestion: maybe `return it == _clustering_row_states.end() ? nullptr : &it->second;`

Reply:
Done.